server/src/main/java/org/candlepin/config/ConfigProperties.java

```diff
@@ -180,21 +177,13 @@ private ConfigProperties() {
         DB_PASSWORD,
     };
 
-    // Pinsetter
-    public static final String TASKS = "pinsetter.tasks";
-    public static final String DEFAULT_TASKS = "pinsetter.default_tasks";
-    public static final String ENABLE_PINSETTER = "candlepin.pinsetter.enable";
-    public static final String PINSETTER_ASYNC_JOB_TIMEOUT = "pinsetter.waiting.timeout.seconds";
-    public static final String PINSETTER_MAX_RETRIES = "pinsetter.retries.max";
-    public static final int PINSETTER_MAX_RETRIES_DEFAULT = 10;
-
     public static final String[] DEFAULT_TASK_LIST = new String[] {};
 
     public static final String MANIFEST_CLEANER_JOB_MAX_AGE_IN_MINUTES =
         "pinsetter.org.candlepin.pinsetter.tasks.ManifestCleanerJob.max_age_in_minutes";
 
     public static final String ENTITLER_JOB_THROTTLE =
-        "pinsetter." + EntitlerJob.class.getName() + ".throttle";
+        "pinsetter.org.candlepin.pinsetter.tasks.EntitlerJob.throttle";
```

Comment:
If we've already changed other job property names, then we no longer have to keep this legacy key. We can specify it using the prefix/suffix pattern (Similarly to what ManifestCleanerJob does; so it would become `candlepin.async.jobs.EntitlerJob.throttle`). For some reason we missed this when porting the EntitlerJob and its constraint.

Reply:
done